l10n_ch_lsv_dd/wizard/export_utils.py

```diff
@@ -46,6 +46,10 @@ def get_treatment_date(prefered_type, line_mat_date, order_sched_date, name,
     elif prefered_type == 'fixed':
         requested_date = fields.Datetime.from_string(order_sched_date) \
             or today
+    elif prefered_type == 'now':
+        requested_date = today
+    else:
+        raise exceptions.Warning('Preferred type not implemented')

```

Comment:
You can wrap the string inside the translate function `_()` so that the error message can be translated to the user language.